examples/block-images.js

```diff
@@ -22,13 +22,7 @@ const puppeteer = require('puppeteer');
 
 const browser = await puppeteer.launch();
 const page = await browser.newPage();
-await page.setRequestInterceptionEnabled(true);
-page.on('request', request => {
-  if (request.resourceType === 'Image')
-    request.abort();
-  else
-    request.continue();
-});
+await page.setBlockedURLs(['.png', '.svg', '.jpg']);
```

Comment:
the old version is actually much more reliable. This version lacks a lot of image formats, e.g. `jpeg` and `webp`

Reply:
I am aware of that. I hit some issues with the interception, it was hitting broken promises when I started to use more than 3 workers. I will do more profiling today to see if I hit the same issue with the setBlockedURLs.

How do you know that 'Network.setBlockedURLs' is deprecated? I only have the tip of the documentation to rely on and for me it is strange that is deprecated since they added this recently to the inspector/network tool (before you add to activate it through the 'press shift 6 times')

Reply:
> How do you know that 'Network.setBlockedURLs' is deprecated? I only have the tip of the documentation to rely on and for me it is strange that is deprecated since they added this recently to the inspector/network tool (before you add to activate it through the 'press shift 6 times')

@hensansi The `Network.setBlockedURLs` is marked as experimental, which means there's no guarantee the method will exist in the upcoming versions of chrome. However, you can safely use the non-experimental methods and domains - we're committed to maintain them.

There's no deprecation process for experimental methods - so there's no way for you to know that this method would be removed. The reason I know is because I work on Chrome DevTools team.

If you absolutely need URL patterns, we've added them as an argument to the `Network.setRequestInterceptionEnabled` method. It's experimental as well, but there are good chances it'll be stabilized.